Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -90,9 +93,18 @@ internal final class SettingsNotificationsViewController: UIViewController {
 
   private func animatePickerView(isHidden: Bool) {
     UIView.animate(withDuration: 0.25) {
-      self.emailPickerViewTopConstraint.constant = isHidden
-        ? 0 : -SettingsNotificationsViewController.emailPickerViewHeight
 
+      if !isHidden {
+        // Tells Voice Over to focus on the picker.
+        self.emailFrequencyPickerView.accessibilityViewIsModal = true
+        UIAccessibility.post(
+          notification: UIAccessibility.Notification.screenChanged,
+          argument: self.emailFrequencyPickerView
+        )
+      }
+
+      self.emailPickerViewTopConstraint.constant = isHidden
```

Comment:
If we use a convenience `height` accessor on a `CGRect` we can fit this on one line like so

```
self.emailPickerViewTopConstraint.constant = isHidden ? 0 : self.emailFrequencyPickerView.frame.height
```

This simply replaces `self.emailFrequencyPickerView.frame.size.height` to `self.emailFrequencyPickerView.frame.height`